app/views/courses/TeacherClassView.coffee

```diff
@@ -171,7 +177,8 @@ module.exports = class TeacherClassView extends RootView
     null
 
   onLoaded: ->
-    @sortedCourses = @classroom.getSortedCourses()
+    @latestCourses = @courses.where({releasePhase: 'released'})
+    utils.sortCourses(@latestCourses)
```

Comment:
(ah, I see now this logic is shared on the server. It'd be great long-term to find a better way than `utils.coffee` to share this kind of model/collection-specific logic)